xlsxwriter v0.3.5 compiling failed

Hi, 
thanks a lot for this amazing repo. 

I'm trying to install it via `cargo install --path .`

this is the error i get on a Apple M2 Pro macOS Ventura 13.4.1:

```
   Compiling xlsxwriter v0.3.5
error[E0412]: cannot find type `size_t` in crate `libxlsxwriter_sys`
    --> /Users/adrianodemarino/.cargo/registry/src/index.crates.io-6f17d22bba15001f/xlsxwriter-0.3.5/src/worksheet.rs:1126:52
     |
1126 |                 buffer.len() as libxlsxwriter_sys::size_t,
     |                                                    ^^^^^^ help: a type alias with a similar name exists: `time_t`
     |
    ::: /Users/adrianodemarino/Downloads/rust_pro/rust-bio-tools-0.42.0/target/release/build/libxlsxwriter-sys-cc1660963e333409/out/bindings.rs:3:6995
     |
3    | ... , stringify ! (_offset))) ; } pub type FILE = __sFILE ; pub type time_t = __darwin_time_t ; pub type lxw_row_t = u32 ; pub type lxw_col_t = u16 ; pub const lxw_boolean_LXW_FALSE ...
     |                                                             --------------- similarly named type alias `time_t` defined here

error[E0412]: cannot find type `size_t` in crate `libxlsxwriter_sys`
    --> /Users/adrianodemarino/.cargo/registry/src/index.crates.io-6f17d22bba15001f/xlsxwriter-0.3.5/src/worksheet.rs:1150:52
     |
1150 |                 buffer.len() as libxlsxwriter_sys::size_t,
     |                                                    ^^^^^^ help: a type alias with a similar name exists: `time_t`
     |
    ::: /Users/adrianodemarino/Downloads/rust_pro/rust-bio-tools-0.42.0/target/release/build/libxlsxwriter-sys-cc1660963e333409/out/bindings.rs:3:6995
     |
3    | ... , stringify ! (_offset))) ; } pub type FILE = __sFILE ; pub type time_t = __darwin_time_t ; pub type lxw_row_t = u32 ; pub type lxw_col_t = u16 ; pub const lxw_boolean_LXW_FALSE ...
     |                                                             --------------- similarly named type alias `time_t` defined here

error[E0063]: missing fields `output_buffer` and `output_buffer_size` in initializer of `lxw_workbook_options`
  --> /Users/adrianodemarino/.cargo/registry/src/index.crates.io-6f17d22bba15001f/xlsxwriter-0.3.5/src/workbook.rs:90:40
   |
90 |             let mut workbook_options = libxlsxwriter_sys::lxw_workbook_options {
   |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `output_buffer` and `output_buffer_size`

Some errors have detailed explanations: E0063, E0412.
For more information about an error, try `rustc --explain E0063`.
error: could not compile `xlsxwriter` (lib) due to 3 previous errors
warning: build failed, waiting for other jobs to finish...
^C  Building [=======================> ] 448/456: librocksdb-sys(build)
```

I tried using the latest version of xlsxwriter (v0.6.0), but I encountered a different error.

```
error[E0599]: no method named `add_worksheet` found for enum `std::result::Result` in the current scope
   --> src/csv/report.rs:131:24
    |
131 |     let mut sheet = wb.add_worksheet(Some("Report"))?;
    |                        ^^^^^^^^^^^^^ method not found in `Result<Workbook, XlsxError>`
    |
note: the method `add_worksheet` exists on the type `xlsxwriter::Workbook`
   --> /Users/adrianodemarino/.cargo/registry/src/index.crates.io-6f17d22bba15001f/xlsxwriter-0.6.0/src/workbook.rs:157:5
    |
157 | /     pub fn add_worksheet<'a>(
158 | |         &'a self,
159 | |         sheet_name: Option<&str>,
160 | |     ) -> Result<Worksheet<'a>, XlsxError> {
    | |_________________________________________^
help: use the `?` operator to extract the `xlsxwriter::Workbook` value, propagating a `Result::Err` value to the caller
    |
131 |     let mut sheet = wb?.add_worksheet(Some("Report"))?;
    |                       +

error[E0599]: no method named `close` found for enum `std::result::Result` in the current scope
   --> src/csv/report.rs:147:8
    |
147 |     wb.close()?;
    |        ^^^^^ method not found in `Result<Workbook, XlsxError>`
    |
note: the method `close` exists on the type `xlsxwriter::Workbook`
   --> /Users/adrianodemarino/.cargo/registry/src/index.crates.io-6f17d22bba15001f/xlsxwriter-0.6.0/src/workbook.rs:277:5
    |
277 |     pub fn close(mut self) -> Result<(), XlsxError> {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: use the `?` operator to extract the `xlsxwriter::Workbook` value, propagating a `Result::Err` value to the caller
    |
147 |     wb?.close()?;
    |       +

For more information about this error, try `rustc --explain E0599`.
error: could not compile `rust-bio-tools` (bin "rbt") due to 2 previous errors
error: failed to compile `rust-bio-tools v0.42.0 (/Users/adrianodemarino/Downloads/rust_pro/rust-bio-tools-0.42.0)`, intermediate artifacts can be found at `/Users/adrianodemarino/Downloads/rust_pro/rust-bio-tools-0.42.0/target`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,7 +40,7 @@ regex = "1.3"
 tera = "1"
 jsonm = "0.1.4"
 chrono = "0.4"
-xlsxwriter = {version = "0.3.5", features= ["use-openssl-md5", "no-md5"]}
+xlsxwriter = {version = "0.6", features= ["use-openssl-md5", "no-md5"]}
 lazy_static = "1.4"
 anyhow = "1"
 thiserror = "1"

--- a/src/csv/report.rs
+++ b/src/csv/report.rs
@@ -15,7 +15,7 @@ use std::io::{Read, Write};
 use std::path::Path;
 use std::str::FromStr;
 use tera::{Context, Tera};
-use xlsxwriter::*;
+use xlsxwriter::prelude::*;
 
 type LookupTable = HashMap<String, HashMap<String, Vec<(String, usize, usize)>>>;
 
@@ -127,7 +127,7 @@ pub(crate) fn csv_report(
         (_, _) => {}
     }
 
-    let wb = Workbook::new(&(output_path.to_owned() + "/report.xlsx"));
+    let wb = Workbook::new(&(output_path.to_owned() + "/report.xlsx"))?;
     let mut sheet = wb.add_worksheet(Some("Report"))?;
     for (i, title) in titles.iter().enumerate() {
         sheet.write_string(0, i.try_into()?, title, None)?;

